modcc/cexpr_render.cpp

```diff
@@ -0,0 +1,112 @@
+#include <ostream>
+#include <unordered_map>
+
+#include "cexpr_render.hpp"
+#include "error.hpp"
+
+void CExprRenderer::render_as_call(const char* sub, Expression* e) {
```

Comment:
Can we use `emit` in place of `render`? Not because `render` is bad, but because we already use `emit` elsewhere in modcc.

Reply:
Should I use `emit` in the class name and wrapper call as well? Easy enough to change.